### PR TITLE
Provide the option to pass the title for an iframe

### PR DESCRIPTION
**Pull request by mvshmakov**
Hey, thanks for the package!

Currently, it is considered a best practice to pass the title for an iframe. The Lighthouse considers this as an issue: https://web.dev/frame-title/. PR provides support for it.

### CODE REVIEW

**Comment by mvshmakov** [comment]
Could you please also make a new version in an NPM when this gets merged?

**Comment by kylemocode** [comment]
> Could you please also make a new version in an NPM when this gets merged?

I will handle this next week because I am in army now...
Thanks for your PR !

**Review by kylemocode — CHANGES_REQUESTED** [review]

File: dist/index.js [hunk 1]
```diff
@@ -1,2 +1,8 @@
 
-module.exports = require('./react-lite-yt-embed.esm.js')
```
[the remark this answers is not in the document]
Comment (mvshmakov) [reply dist/index.js:2]: Done, thanks!

**Review by kylemocode — APPROVED** [review]

**Comment by kylemocode** [comment]
> Hey, thanks for the package!
> 
> Currently, it is considered a best practice to pass the title for an iframe. The Lighthouse considers this as an issue: https://web.dev/frame-title/. PR provides support for it.

@mvshmakov Now the latest version is @1.2.7

**Pull request merged**